### DISCUSSION TITLE
Container initialization failed as it has already been initialized with a different share scope

### DISCUSSION THREAD

**Discussion by xuebigshuai** [webpack/webpack#16514] [Q&A] [answered]
# Bug report

**What is the current behavior?**
when app A and  app B   use common library C  by MF, and  A use B by MF，config shared。throw Error  'Container initialization failed as it has already been initialized with a different share scope'

**If the current behavior is a bug, please provide the steps to reproduce.**
A use B  by MF;  
A use C by MF;  
B use C  by MF；

config ModuleFederationPlugin shared  property 

**What is the expected behavior?**
application A  can use 
 

**Other relevant information:**
webpack version:  5.73.0
Node.js version:  16.14.0
Operating System: window 10
Additional tools:

**Comment by xuebigshuai** [comment]
shared: {
            vue: {
              eager: false,
              singleton: true
            }
 }

**Comment by xuebigshuai** [comment]
I find  webpack Project  don't have the problem.   It's not good in vite  project

**Reply by alexander-akait** [reply, marked as answer]
Do you build your application using webpack and vite together?

**Reply by xuebigshuai** [reply]
hostApp use webpack  another remoteApp use webpack

**Reply by MirKml** [reply]
OT: It's nice to see you are back Sasha @alexander-akait, save and sound. I think last 2 years were very tough for you.
Wish you best for next years, hope for better world.
